package.json

```diff
@@ -9,8 +9,8 @@
   },
   "scripts": {
     "build": "run-p -s build:*",
-    "build:clean": "del packages/openapi-typescript/dist && del packages/openapi-fetch/dist",
-    "build:ts": "tsc -b packages/openapi-typescript/tsconfig.build.json packages/openapi-fetch/tsconfig.build.json",
```

Comment:
`tsc -b` is a nice little optimization, but when we’re only building 2 packages, this won’t have much impact at all. Just let each package decide how it builds, even if it’s a second or two slower.